docs/compression_algorithms/post_training/ONNX.md

```diff
@@ -0,0 +1,24 @@
+## Post-Training Quantization for ONNX
+
+NNCF supports [ONNX](https://onnx.ai/) backend for the Post-Training Quantization algorithm.
+This guide contains some notes that you have to consider before working with NNCF for ONNX.
```

Comment:
```suggestion
This guide contains requirements to consider before working with NNCF for ONNX.
```

Reply:
This is not only requirements. There are also advices.

Reply:
"have to" is an obligation/restriction, maybe "should"?

Reply:
Some requirements?

Reply:
requirements and recommendations - could work

Reply:
Done